waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -184,31 +179,21 @@ isDbConnectionPossible DbConnectionSuccess = True
 isDbConnectionPossible DbNotCreated = True
 isDbConnectionPossible _ = False
 
-generatePrismaClients :: Path' Abs (Dir ProjectRootDir) -> IO (Either String ())
-generatePrismaClients projectRootDir = do
-  generateResult <- liftA2 (>>) generatePrismaClientForServer generatePrismaClientForWebApp projectRootDir
-  when (isRight generateResult) updateDbSchemaChecksumOnLastGenerate
-  return generateResult
-  where
-    generatePrismaClientForServer = generatePrismaClient serverPrismaClientOutputDirEnv J.Server
-    generatePrismaClientForWebApp = generatePrismaClient webAppPrismaClientOutputDirEnv J.WebApp
-    updateDbSchemaChecksumOnLastGenerate =
-      writeDbSchemaChecksumToFile projectRootDir dbSchemaChecksumOnLastGenerateFileProjectRootDir
-
-generatePrismaClient ::
-  (String, String) ->
-  J.JobType ->
-  Path' Abs (Dir ProjectRootDir) ->
-  IO (Either String ())
-generatePrismaClient prismaClientOutputDirEnv jobType projectRootDir = do
+generatePrismaClient :: Path' Abs (Dir ProjectRootDir) -> IO (Either String ())
+generatePrismaClient projectRootDir = do
```

Comment:
Since we're only generating a single client, this function is now much shorter.

Reply:
Sweet